cpp/src/groupby/sort/quantiles/quantiles.hpp

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
this is also for general quantiles computation. not needed to be in groupby directory.